textile/fabric_pretreatment/doctype/pretreatment_order/pretreatment_order.py

```diff
@@ -23,7 +23,6 @@
 
 force_fields = force_customer_fields + force_fabric_fields + force_process_component_fields
 
-
```

Comment:
Minor: Do not remove this line. We keep double new lines between non object methods